packages/lab/src/pill/PillNext.tsx

```diff
@@ -0,0 +1,186 @@
+import {
+  forwardRef,
+  MouseEvent,
+  PropsWithChildren,
+  KeyboardEvent,
+  useState,
+  ComponentPropsWithoutRef,
+} from "react";
+import { useWindow } from "@salt-ds/window";
+import { CloseIcon, CloseSmallIcon } from "@salt-ds/icons";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import { makePrefixer, useButton, useDensity } from "@salt-ds/core";
+import pillCss from "./PillNext.css";
+import clsx from "clsx";
+
+type ClickEvent =
+  | MouseEvent<Element, globalThis.MouseEvent>
+  | KeyboardEvent<HTMLDivElement>;
+
+export interface PillProps extends ComponentPropsWithoutRef<"div"> {
+  disabled?: boolean;
+  className?: string;
+  icon?: React.ReactNode;
```

Comment:
this could come from imports
```suggestion
  icon?: ReactNode;
```